network_observability/network-observability-overview.adoc

```diff
@@ -37,7 +37,41 @@ In *Observe* -> *Dashboards*, the *Netobserv* dashboard provides a quick overvie
  * *Top byte rates received per source and destination workloads*
 
 *Infrastructure* and *Application* metrics are shown in a split-view for namespace and workloads.
-You can configure the `FlowCollector` `spec.processor.metrics` to add or remove metrics by changing the `ignoreTags` list. For more information about available tags, see the xref:../network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference]
+You can configure the `FlowCollector` custom resource `spec.processor.metrics.includeList` to add or remove metrics.
+
+Available metrics names are as follows (names followed by * are enabled by default): 
+
+* namespace_egress_bytes_total
```

Comment:
Would it make sense to list all these guys in `monospace` font?
```suggestion
* `namespace_egress_bytes_total`
```

Reply:
I think it would, yea. Bc they will all be specified in a Prometheus definition file. Good catch. Also FYI I am moving this content over to another PR: https://github.com/openshift/openshift-docs/pull/69106/files .

Reply:
I'll incorporate this feedback over there. TY!